### ISSUE TITLE
Replication Package

### ISSUE THREAD

**Issue by jssmith1** [DeveloperLiberationFront/iTrustInterviews#417]
Editor:

>R2 and R3 also felt the work needed more replicability along multiple dimensions. For example, R2 was concerned that the data cited in citation [33] was hosted on a non-archival student website. I recommend, if feasible, incorporating the questions in an appendix in the draft, or if the length of the content is prohibitive, finding a way to provide at least a sampling of the questions as an appendix. R3 also found the attack tree analysis inadequately specified, and sought a more detailed explanation of how the analysis was conducted so that others might verify or replicate the same analysis in future work.

**Comment by jssmith1** [comment]
R2:

>* The results (the concrete questions and strategies) are published online on one of the authors institution web sites and are linked from the article. Since the mentioned author is a Ph.D. student I am afraid that the page will vanish in a year or two (similar to the results of the original paper that are already offline). Is there a way to make the results more long-lasting? Since the results of the conference paper are not reachable anymore it is not possible to identify whether presented strategies were already in the data from the conference paper. This fact makes it harder to judge  if the paper is novel.

**Comment by jssmith1** [comment]
- [x] Added strategies to figshare (still need to be published and referenced in the paper):
https://figshare.com/account/home#/projects/24439

**Comment by jssmith1** [comment]
- [x] Ensure consistent naming between paper and question categories

**Comment by jssmith1** [comment]
We apologize that the material from the original submission was not available and agree that the material should be in a longer term archival location. Based on R2's suggestion, we added the questions to the appendix. The strategy analysis results span 42 pages, so we opted to exclude them from the appendix. Additionally, we moved all of the study materials, including the strategy results, to an archival website (https://figshare.com/projects/How_Developers_Diagnose_Potential_Security_Vulnerabilities_with_Static_Analysis/24439).
